assets/js/state/selectors/lastExecutions.test.js

```diff
@@ -2,7 +2,7 @@ import {
   hostFactory,
   clusterFactory,
   catalogCheckFactory,
-  checksExecutionCompletedFactory,
+  checksExecutionCompletedForTargets,
```

Comment:
```suggestion
  checksExecutionCompletedForTargetsFactory,
```

:D

Reply:
Ok, I am gonna make it a real factory then